.github/workflows/ci.yml

```diff
@@ -42,16 +46,25 @@ jobs:
         sudo -u postgres /usr/lib/postgresql/${version}/bin/psql -U postgres -c "CREATE ROLE repl WITH LOGIN REPLICATION PASSWORD 'secretpassword';"
     - name: GCC/mkdir
       run: mkdir build
-      working-directory: /home/runner/work/pgmoneta/pgmoneta_ext/
+      working-directory: /home/runner/work/pgmoneta_ext/pgmoneta_ext/
     - name: GCC/cmake
       run: export CC=/usr/bin/gcc && cmake -DCMAKE_BUILD_TYPE=Debug ..
-      working-directory: /home/runner/work/pgmoneta/pgmoneta_ext/build/
+      working-directory: /home/runner/work/pgmoneta_ext/pgmoneta_ext/build/
     - name: GCC/make
       run: make
-      working-directory: /home/runner/work/pgmoneta/pgmoneta_ext/build/
+      working-directory: /home/runner/work/pgmoneta_ext/pgmoneta_ext/build/
     - name: rm -Rf
       run: rm -Rf build/
-      working-directory: /home/runner/work/pgmoneta/pgmoneta_ext/
+      working-directory: /home/runner/work/pgmoneta_ext/pgmoneta_ext/
+    - name: Create extension
+      run: |
+        echo ${{ secrets.USERPWD }} | sudo -S -u postgres /usr/lib/postgresql/${version}/bin/psql -U postgres -c "DROP EXTENSION IF EXISTS pgmoneta_ext;"
```

Comment:
Have you tried just running psql, and not as `postgres` user? I think with postgres installed you could probably run postgres even not as `postgres` user. This way you don't need `sudo -u postgres`, and thus bypassing this nasty password problem.

Reply:
It's a brilliant concept. We can successfully execute `postgresql` without the need for `sudo`. It functions admirably.